contrib/charts/cert-manager/templates/letsencrypt_cluster_issuer.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if and .Release.IsUpgrade .Values.letsencryptEmail }}
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-production
+spec:
+  acme:
+    server: https://acme-v01.api.letsencrypt.org/directory
+    email: {{ .Values.letsencryptEmail }}
+    privateKeySecretRef:
+      name: letsencrypt-production-cluster-key-pair
+    http01: {}
+---
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-staging
+spec:
+  acme:
+    server: https://acme-staging.api.letsencrypt.org/directory
+    email: {{ .Values.letsencryptEmail }}
+    privateKeySecretRef:
+      name: letsencrypt-staging-cluster-key-pair
```

Comment:
Given that this feature is specifically targeted at those coming from tools like kube-lego, it is actually incompatible with the "Migrating from kube-lego doc" right now. In that doc, we instruct users how to recover their ACME private key so that they can create an Issuer that uses that key. Should the "migrating from kube-lego" doc be updated to utilise this new functionality in the Helm chart?

Whilst it might make it quicker for users to get started, it'd be a shame to lose the opportunity to expose users to resources like Issuers and ClusterIssuers, as its probably the most foreign part to a new user (and imo, we shouldn't try and hide it just because it's different to what they are used to)